src/v2/Components/TextLink.tsx

```diff
@@ -36,8 +36,8 @@ export class TextLink extends React.Component<LinkProps, null> {
 
 const StyledTextLink = styled(TextLink)`
   ${garamond("s15")};
-  color: ${(props) => props.color};
-  text-decoration: ${(props) => (props.underline ? "underline" : "none")};
+  color: ${props => props.color};
```

Comment:
I'm kinda surprised by this, I thought prettier was always adding parens now... Running prettier over the file locally doesn't change it though. 🤷